### ISSUE TITLE
checkout array indexing

### ISSUE THREAD

**Issue by RiyazAina-DeepML** [ecaradec/humble-yolo#16]
thanks for your repository. it help a lot.
but in your main.py code file i think there is one correction if i am not wrong.

your code contains
`    # first three values are classes : cat, rat, and none.
    # However yolo doesn't predict none as a class, none is everything else and is just not predicted
    # so I don't use it in the loss
    y_true_class = y_true[...,0:2]
    y_pred_class = y_pred[...,0:2] `
but you mentioned that there are three classes in your dateset.
there must 3 insted of 2 in array indexing.
please let me know whether it is correct or not.

**Comment by ecaradec** [comment]
It should be 2 :
None shouldn't be considered a class. In classical YOLO None should mean anything that the network doesn't know? Because it should represent a different set of things, it is not a very good class. 
However in my training set, it is very possible to train the network to learn None as a class, because here None means empty (as only black pixels ). It also helps the network training much faster. It works in this particular case, but it not the general case.
What you want is that the network classify every box with the highest probability element and have the extra 'conf_loss' that is trained to tell if there is something in this box or not.
